app/views/validation_requests/_additional_document_validation_requests.html.erb

```diff
@@ -1,15 +1,17 @@
-<li>
-  <h2 class="moj-task-list__section">
-    <span class="moj-task-list__section-number"><%= count_total_requests(@validation_requests, "additional_document_validation_requests") %> </span>Provide new or missing documents
-  </h2>
-  <ul class="moj-task-list__items" id="additional-document-validation-requests">
+<li class="govuk-!-margin-top-8">
```

Comment:
Do we need these outer `<li>`s around the inner `<ul>` and `<li>`? The report refers to lists with only one item, but I don't really know enough about how the task list component works to figure out if it can be simplified.

Reply:
Confusingly that's how the component works. It's similar to what we've got in the BOPS assessment tasklist, but with numbers